core/src/Streamly/Internal/Data/Stream/Expand.hs

```diff
@@ -627,6 +626,7 @@ concatPairsWith par f m =
 -- It can be used to traverse a tree structure.  For example, to list a
 -- directory tree:
 --
+-- >> import qualified Streamly.Internal.FileSystem.Dir as Dir
```

Comment:
Need to use `>>>` for docspec to work.

Reply:
I know. But it won't as `Dir` is not in `streamly-core`. This change should be reverted once your PRs are merged.